frontend2/src/views/Account.tsx

```diff
@@ -59,6 +56,11 @@ const Account: React.FC = () => {
     await uploadResume.mutateAsync({ resume: data.file[0] });
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  const onResumeDownload: EventHandler<React.MouseEvent<HTMLButtonElement>> = async () => {
+    await downloadResume.mutateAsync({ id: user?.id ?? 0 });
```

Comment:
nit: prefer disabling the button if user isn't loaded, as I'm not sure if our db would successfully reject for a request to id 0 by default. Then we could block the event via a if conditional if not loaded, lmk if this makes sense.